Neos.Flow/Classes/SignalSlot/SignalInformation.php

```diff
@@ -0,0 +1,60 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\Flow\SignalSlot;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+/**
+ * A SignalInformation instance carries information about the signal that arrives
+ * at a slot during a signal/slot dispatch operation.
+ *
+ * @api
+ */
+class SignalInformation
```

Comment:
We can even make this final I think (and maybe also proxy=false?)
```suggestion
final class SignalInformation
```